pkg/features/list.go

```diff
@@ -98,4 +98,7 @@ var (
 	SensorAggregateDeploymentReferenceOptimization = registerFeature("Enables a performance improvement by aggregating deployment references when the same reference is queued for processing", "ROX_AGGREGATE_DEPLOYMENT_REFERENCE_OPTIMIZATION", true)
 	// ACSCSEmailNotifier enables support for the ACSCS email notifier integratioon
 	ACSCSEmailNotifier = registerFeature("Enable support for ACSCS Email notifier type", "ROX_ACSCS_EMAIL_NOTIFIER", false)
+
+	// RuntimeFilters enables filtering of runtime events.
+	RuntimeFilters = registerFeature("Enable runtime event filtering", "ROX_RUNTIME_FILTERS", false)
```

Comment:
Change this and all usages of filter to something like `ROX_COLLECTOR_RUNTIME_CONFIG`.